### PR TITLE
Add custom rule label and 3DS result input

### PR DESCRIPTION
**Pull request by kevcenteno**
* Adds custom rule label to disposition object
* Adds 3DS result to credit card input
* Updates disposition.action docs to include "test" disposition

### CODE REVIEW

**Comment by horgh** [comment]
Lemme know when it's ready for review!

**Comment by kevcenteno** [comment]
Ready for review :smile_cat:

File: lib/minfraud/components/credit_card.rb [hunk 1]
```diff
@@ -62,17 +62,28 @@ class CreditCard < Base
       # @return [String, nil]
       attr_accessor :cvv_result
 
+      # Whether the outcome of 3-D Secure verification (e.g. Safekey,
+      # SecureCode, Verified by Visa) was successful. `true` if customer
+      # verification was successful, or `false` if the customer failed
```
Comment (horgh) [inline lib/minfraud/components/credit_card.rb:67]: I believe markdown does not work in these comments. `+false+` would. (Actually I think yard can be made to support markdown but I don't think ours is set up to do that? I'm not sure. Either way it is not working AFAICT).

File: CHANGELOG.md [hunk 2]
```diff
@@ -5,6 +5,9 @@
 * Adds new processor to `Minfraud::Components::Payment`: `:cardknox`,
   `:creditguard`, `:credorax`, `:datacap`, `:dlocal`, `:onpay`, and
   `:safecharge`.
+* Adds `rule_label` to minFraud output `/disposition`.
+* Adds `was_3d_secure_successful` attribute to
+  `Minfraud::Components::CreditCard`.
```
Comment (horgh) [inline CHANGELOG.md:10]: These could probably benefit from including a bit more information about them. For example, "Adds support for the /credit_card/was_3d_secure_successful input. This is available by setting the was_3d_secure_successful attribute on Minfraud::Components::CreditCard" or something like that.

File: lib/minfraud/components/credit_card.rb [hunk 3]
```diff
@@ -62,17 +62,28 @@ class CreditCard < Base
...
+      # verification. If 3-D Secure verification was not used, was unavailable,
+      # or resulted in another outcome other than success or failure, do not
```
Comment (horgh) [inline lib/minfraud/components/credit_card.rb:69]: "an outcome other than success or failure" perhaps?

**Review by horgh — CHANGES_REQUESTED** [review]

**Comment by kevcenteno** [comment]
@horgh Ready for another round of review :mag:

**Review by horgh — APPROVED** [review]

**Pull request merged**